corporate/lib/stripe.py

```diff
@@ -1595,7 +1594,6 @@ def generate_stripe_invoice(
             "license_management": license_management,
             "price_per_license": None,
             "fixed_price": None,
-            "seat_count": seat_count,
```

Comment:
I guess this `seat_count` is included in the metadata we send to Stripe, but it also looks like a duplicate of `licenses`? OK.